src/Form/Form/RevisionJsonMenuNestedType.php

```diff
@@ -32,40 +32,39 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
     {
         /** @var FieldType $fieldType */
         $fieldType = $options['field_type'];
+        /** @var ContentType $contentType */
+        $contentType = $options['content_type'];
 
```

Comment:
```suggestion
        $contentType = $options['content_type'] ?? null;
        
        if (!$contentType instanceof ContentType) {
            throw new \RuntimeException('Unexpected ContentType object');
        }

```

Reply:
The option resolver has a allowType set on ContentType, so it will always be a ContentType instance